source/docs/v4/client-initialization.md

```diff
@@ -19,10 +19,15 @@ In the examples below, the `io` object comes either from:
 - NPM
 
 ```js
-// ES6 import or TypeScript
+// CommonJS strict
+const { io } = require("socket.io-client");
+// or
 import { io } from "socket.io-client";
-// CommonJS
+
+// NodeJS
 const io = require("socket.io-client");
+// or
+import io from "socket.io-client";
```

Comment:
Not sure if it really simplifies the code example :smile: 

Both `const { io } = require("socket.io-client");` and `const io = require("socket.io-client");` should work, we've kept the latter for backward-compatibility.

Reply:
I'm using ES modules and the named import didn't work (`import { io } from "socket.io-client"`), hence my confusion. Perhaps I took the example too literally, or should've known to try the default import...I can close this